src/views/open-api-docs-view/server.ts

```diff
@@ -109,10 +109,9 @@ export async function getStaticProps({
 		typeof sourceFile === 'string'
 			? sourceFile
 			: await fetchGithubFile(sourceFile)
-	const schemaData = await parseAndValidateOpenApiSchema(
-		schemaFileString,
-		massageSchemaForClient
-	)
+	const schemaData = await parseAndValidateOpenApiSchema(schemaFileString, [
+		massageSchemaForClient,
+	])
```

Comment:
Note this is the _existing_ open-api-docs-view, _not_ the new "split-operations" one. This change reflects the updated interface for `parseAndValidateOpenApiSchema`, which now supports an _array_ of schema transform functions, rather than a single function.